### ISSUE TITLE
weboob's site-package python module is not loadable

### ISSUE THREAD

**Issue by wrobelda** [Homebrew/homebrew-core#67334]
# Bug report

The weboob formula installs the python module (`/usr/local/Cellar/weboob/2.0_2/libexec/lib/python3.9/site-packages/`), however the module itself cannot be loaded using brew-provided python3 environment:

```
>>> import weboob
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'weboob'
```

Based on the [guidelines](https://github.com/Homebrew/brew/blob/master/docs/Python-for-Formula-Authors.md#installing-1), my understanding is that the package would need to install a `.pth` file into the `prefix` site-packages for this to work. Please consider doing so, as any software (e.g. KMyMoney) that uses the weboob API most likely depends on being able to import the `weboob` python module.

## Step-by-step reproduction instructions (by running `brew install` commands)
`brew install weboob`
`python3 -c "import weboob;"`

**Comment by carlocab** [comment]
>the package would need to install a `.pth` file into the `prefix` site-packages for this to work

Please open a PR for this. Thanks!

**Comment by carlocab** [comment]
Closed in ad31d73cb79df62b182ea5a85e3cc8e1a0039e8c.
